plasmapy/formulary/collisions.py

```diff
@@ -2075,15 +2075,15 @@ def _v_0(self):
         return (
             4
             * math.pi
-            * self.species[0].charge ** 2
-            * self.species[1].charge ** 2
+            * (self.species[0].charge_number * e.esu) ** 2
```

Comment:
Tips on how to make this less ugly would be appreciated. Shouldn't functions only take MKS units?

Reply:
@namurphy 

To elaborate, the problem here is that the NRL formulary (from which we are adapting these functions) is in CGS, but we want to take MKS input for PlasmaPy.

Reply:
@JaydenR2305 I don't think you want the charge number here, I think this should just be `self.species[0].charge.to(u.statC)`

Looks like u.esu is equivalent to u.statC